nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/nar/NarClassLoaders.java

```diff
@@ -224,7 +228,7 @@ private InitContext load(final ClassLoader rootClassloader,
                     // see if this class loader is eligible for loading
                     ClassLoader narClassLoader = null;
                     if (narDependencyCoordinate == null) {
-                        narClassLoader = createNarClassLoader(narDetail.getWorkingDirectory(), jettyClassLoader);
+                        narClassLoader = createNarClassLoader(narDetail.getWorkingDirectory(), rootClassloader);
```

Comment:
I think we should make this line consistent with the logic below around line 390...

In this case we are saying, if no dependency then the parent is always the root class loader.

In the other case we are saying, if no dependency and if jetty bundle exists (which currently it always does), then make jetty bundle the parent, otherwise the parent is null.

I think both cases should probably make the parent be the jetty class loader if the jetty bundle exists, otherwise the root class loader.